repos/system_upgrade/common/actors/removeleftoverpackages/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import removeleftoverpackages
+from leapp.models import LeftoverPackages, RemovedPackages
+from leapp.tags import ExperimentalTag, IPUWorkflowTag, RPMUpgradePhaseTag
+
+
+class RemoveLeftoverPackages(Actor):
+    """
+    Remove packages left on the system after the upgrade to higher major version of RHEL.
+
+    Removal of packages is necessary in order to keep the machine in supported state.
+    Actor generates report telling users what packages have been removed.
```

Comment:
Since you removed the report this should also be deleted.

Reply:
Looking at the reportleftoverpackages actor, I realize this actor generates a message which that one then consumes and generates the report. This could be briefly described here.

Reply:
Improved the description!